patch-db/src/store.rs

```diff
@@ -216,24 +215,34 @@ impl Store {
         let patch_bin = serde_cbor::to_vec(&*patch)?;
         let persistent_undo = json_patch::patch(&mut self.persistent, &*patch)?;
         self.revision += 1;
-        async {
-            let file_len = self.file.metadata().await?.len();
-            if file_len != self.file_cursor {
-                self.file.set_len(self.file_cursor).await?;
-                self.file.seek(SeekFrom::Start(self.file_cursor)).await?;
+        if let Err(_e) = if self.revision % 4096 == 0 {
```

Comment:
Still a big number, why this number

Reply:
that's how often we compress automatically